packages/forms/frontend/sirius-components-forms/src/propertysections/PropertySectionLabel.tsx

```diff
@@ -22,6 +22,7 @@ const usePropertySectionLabelStyles = makeStyles((theme) => ({
     display: 'flex',
     flexDirection: 'row',
     alignItems: 'center',
+    marginRight: theme.spacing(2),
```

Comment:
A component cannot make hypothesis on the location where it will be used. Perform the layout on the parent not the children components. In this case, I suspect that a `gap: theme.spacing(2)` is missing in the places where you have added a `display: 'flex', flexDirection: 'row'`

Reply:
Done. But in order to avoid getting the weird empty space when button or label widgets are used without a LabelExpression

![image](https://github.com/eclipse-sirius/sirius-web/assets/4439675/60169810-b41a-4a7e-88bd-517ac58a69bc)

I also modified the PropertySectionLabel to return a *null* component when it has nothing to display.


```typescript
export const PropertySectionLabel = ({ editingContextId, formId, widget, subscribers }: PropertySectionLabelProps) => {
  const classes = usePropertySectionLabelStyles();

  if (!widget.label && subscribers.length == 0 && !widget.hasHelpText) {
    return null;
  } else {
    return (
      <div className={classes.propertySectionLabel}>
        <Typography className={classes.typography} variant="subtitle2">
          {widget.label}
        </Typography>
        {widget.hasHelpText ? (
          <HelpTooltip editingContextId={editingContextId} formId={formId} widgetId={widget.id} />
        ) : null}
        <div className={classes.subscribers}>
          {subscribers.map((subscriber) => (
            <Tooltip title={subscriber.username} arrow key={subscriber.username}>
              <Avatar classes={{ root: classes.avatar }}>{subscriber.username.substring(0, 1).toUpperCase()}</Avatar>
            </Tooltip>
          ))}
        </div>
      </div>
    );
  }
};
```

What do you thinlk about that? It means that this behavior will also apply on all other widgets. For example for a TextField with no LabelExpression, NoSubscribers and No Help Expression

![image](https://github.com/eclipse-sirius/sirius-web/assets/4439675/4b842573-0124-4f96-bfcf-666c3724a4c7)